src/index.ts

```diff
@@ -143,13 +144,13 @@ async function getBooleanWithPrompt(prompt, options){
     return response;
 }
 
-function getUnknownKey(obj, knownKeys){
+function getUnknownKey(obj:ExpectTypeParams, knownKeys: string[]){
```

Comment:
Type is likely too strict, I would change it to a looser:

```suggestion
function getUnknownKey(obj: Record<string, unknown>, knownKeys: string[]){
```